docs/conf.py

```diff
@@ -110,7 +99,9 @@ def linkcode_resolve(domain, info):
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'api/modules.rst']
+exclude_patterns = [
+    '_build', 'Thumbs.db', '.DS_Store',
+    'api/modules.rst', 'api/smriprep.rst']
```

Comment:
@rwblair I've found this easier than modifying apidoc templates to achieve a better rendering of the API (please check the corresponding ``docs/api.rst`` file).

Reply:
Makes sense, creating the api.rst directly is much more straightforward since its the only place the template is used and we aren't adding new submodules to be listed there that often.